src/main/java/fmfi/dalekohlad/Modules/Axis.java

```diff
@@ -130,25 +134,15 @@ public void SlewDE() {
     }
 
     public void update(JsonObject jo) {
-        if(jo.get("PAEncoder") != null) Platform.runLater(() -> {info_polar[0].setText(jo.get("PAEncoder").getAsString());});
-
-        if(jo.get("PAHAApparent") != null) Platform.runLater(() -> {info_polar[1].setText(jo.get("PAHAApparent").getAsString());});
-
-        if(jo.get("PAHARAJ2000") != null) Platform.runLater(() -> {info_polar[2].setText(jo.get("PAHARAJ2000").getAsString());});
-
-        if(jo.get("PAAzimuth") != null) Platform.runLater(() -> {info_polar[3].setText(jo.get("PAAzimuth").getAsString());});
-
-        if(jo.get("PAStatus") != null) Platform.runLater(() -> {info_polar[4].setText(jo.get("PAStatus").getAsString());});
-
-        if(jo.get("DEEncoder") != null) Platform.runLater(() -> {info_declination[0].setText(jo.get("DEEncoder").getAsString());});
-
-        if(jo.get("DEApparent") != null) Platform.runLater(() -> {info_declination[1].setText(jo.get("DEApparent").getAsString());});
-
-        if(jo.get("DEDEJ2000") != null) Platform.runLater(() -> {info_declination[2].setText(jo.get("DEDEJ2000").getAsString());});
-
-        if(jo.get("DEElevation") != null) Platform.runLater(() -> {info_declination[3].setText(jo.get("DEElevation").getAsString());});
-
-        if(jo.get("DEStatus") != null) Platform.runLater(() -> {info_declination[4].setText(jo.get("DEStatus").getAsString());});
+        for(int i = 0; i < 5;i++) {
```

Comment:
vsimol som si, ze tento for sa opakuje a asi bude uzitocny aj pri dalsich moduloch, mozno by stalo za to dat ho ako funkciu do GUIModule

Reply:
na https://github.com/TIS2020-FMFI/dalekohlad/blob/31ef738f11d0bb7c372c8cdd1c9133bf32d8b669/src/main/java/fmfi/dalekohlad/Modules/Axis.java#L44 som si inak este vsimol, ze chyba runlater